crates/rspack_core/src/exports_info.rs

```diff
@@ -1829,3 +1829,247 @@ macro_rules! debug_exports_info {
     }
   };
 }
+
+pub trait ModuleGraphAccessor<'a> {
```

Comment:
Adding module graph accessors to prevent module graph variability issue. Use immutable accessor during code generation. Use mutable accessor during tree shaking plugins